packages/core/src/runtime/resolveResetStyleRules.ts

```diff
@@ -0,0 +1,122 @@
+import hashString from '@emotion/hash';
+import { convertProperty } from 'rtl-css-js/core';
+
+import { RESET_HASH_PREFIX } from '../constants';
+import { GriffelStyle, GriffelResetStyle } from '../types';
+import { compileCSSRules, normalizePseudoSelector } from './compileCSS';
+import { isMediaQuerySelector } from './utils/isMediaQuerySelector';
+import { isLayerSelector } from './utils/isLayerSelector';
+import { isNestedSelector } from './utils/isNestedSelector';
+import { isSupportQuerySelector } from './utils/isSupportQuerySelector';
+import { isObject } from './utils/isObject';
+import { hyphenateProperty } from './utils/hyphenateProperty';
+
+/**
+ * @internal
+ */
+export function createStringFromStyles(styles: GriffelResetStyle) {
```

Comment:
This function is currently private to this file. Why is it exported?

Reply:
No reason, I covered it with tests initially, but then switched to `resolveResetStyleRules`.